awx/settings/defaults.py

```diff
@@ -125,10 +125,10 @@ def IS_TESTING(argv=None):
 # Absolute filesystem path to the directory for job status stdout (default for
 # development and tests, default for production defined in production.py). This
 # directory should not be web-accessible
-JOBOUTPUT_ROOT = os.path.join(BASE_DIR, 'job_output')
+JOBOUTPUT_ROOT = '/var/lib/awx/job_status/'
 
 # Absolute filesystem path to the directory to store logs
-LOG_ROOT = os.path.join(BASE_DIR)
+LOG_ROOT = '/var/log/tower/'
 
 # The heartbeat file for the tower scheduler
 SCHEDULE_METADATA_LOCATION = os.path.join(BASE_DIR, '.tower_cycle')
```

Comment:
Can we change this (and others) and effectively remove BASE_DIR entirely?

Reply:
`BASE_DIR` is where the actual AWX code lives. In the development environment, this location (`/awx_devel`) is different from the `/var/lib` locations. I think its naming is poor, but I don't think we can get rid of it easily.